bin/update/01-run.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+RELEASE=$1
+UMBREL_ROOT=$2
+UMBREL_USER=$3
+
+echo
+echo "======================================="
+echo "============= OTA UPDATE =============="
+echo "======================================="
+echo "=========== Stage: Install ============"
+echo "======================================="
+echo
+
+cat <<EOF > "$UMBREL_ROOT"/statuses/update-status.json
+{"state": "installing", "progress": 33, "description": "Configuring settings", "updateTo": "$RELEASE"}
+EOF
+
+# Checkout to the new release
+cd "$UMBREL_ROOT"/.umbrel-"$RELEASE"
+
+# Update RPC Password in docker-compose.yml
+# Get gnu sed
+gnused=sed
+if [[ "$(uname)" == "Darwin" ]]; then
+  if ! command -v gsed >/dev/null 2>&1; then
+    echo "Error: This script requires gnu-sed!"
+    echo "Install it with:"
+    echo "  brew install gnu-sed"
+    exit 1
+  fi
+  gnused=gsed
+fi
+
+echo "Updating RPC Password in docker-compose.yml"
+RPCPASS=$(cat "$UMBREL_ROOT"/secrets/rpcpass.txt)
+$gnused -i "s/RPCPASS/${RPCPASS}/g;" docker-compose.yml
+
+# echo "Setting regtest"
+# $gnused -i 's/mainnet/regtest/g; ' docker-compose.yml
+# $gnused -i "s/RPCPORT/18443/g;" docker-compose.yml
+
+echo "Setting mainnet"
```

Comment:
Not worth blocking the PR over this but it would be good to not have any network settings hardcoded.

The updater should detect the existing network (mainnet/testnet/regtest) and apply the appropriate updates.

That way we can run OTA updates in development for testnet/regtest instances.

Reply:
In fact this should probably not be done as part of the updater at all.

It should be handled by the configuration scripts. If the config scripts do too much then we can split it up into smaller parts and only call the part we need from the updater.

Again, not worth blocking the PR for. We can improve the update scripts in future PRs.

Reply:
I agree that this isn't something that the update scripts should touch. We can look into doing better config handling post merge.